bridge-impl/src/main/java/com/liferay/faces/bridge/component/internal/ComponentUtil.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2000-2016 Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.liferay.faces.bridge.component.internal;
+
+import javax.faces.component.UIComponent;
+import javax.faces.component.ValueHolder;
+
+
+/**
+ * @author  Kyle Stiemann
+ */
+public final class ComponentUtil {
+
+	private ComponentUtil() {
+		throw new AssertionError();
+	}
+
+	public static String getComponentValue(UIComponent uiComponent) {
```

Comment:
@stiemannkj1 Agreed, `getComponentValue()` should be removed from ResourceUtil. Please send a follow-up pull request. Thanks.